documentation/book/ref-kafka-bridge-producer-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-producer-configuration-{context}']
+= Kafka Bridge Producer configuration
+
+Kafka Bridge producer is configured using the properties in `KafkaBridge.spec.producer`.
+This property contains the Kafka Bridge producer configuration options as keys.
+The values can be one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+You can specify and configure the options listed in the {ApacheKafkaProducerConfig}.
```

Comment:
All options

Reply:
I believe yes. @ppatierno ?

Reply:
I would say yes, but avoid to specify "all" for now :-)